dev_guide/getting_traffic_into_cluster.adoc

```diff
@@ -0,0 +1,147 @@
+[[getting-traffic-into-cluster]]
+= Getting Traffic Into The Cluster
+{product-author}
+{product-version]
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+ifdef::openshift-origin,openshift-enterprise,openshift-dedicated[]
+== Overview
+There are many ways to access the cluster. This section describes some
+commonly used approaches.
+
+The recommendation is:
+
+- If you have HTTP/HTTPS -- Use the xref:../dev_guide/getting_traffic_into_cluster.adoc#using-a-router[Router],
+- If you have Encrypted TCP (i.e., TLS with the SNI header) -- Use the xref:../dev_guide/getting_traffic_into_cluster.adoc#using-a-router[Router],
+- If you have TCP or UDP -- Use xref:../dev_guide/getting_traffic_into_cluster.adoc#using-the-loadbalancer[LoadBalancer],
```

Comment:
This is the previous comment:

Perhaps we can make a table comparing features now we have 5 options.  Or a flow chart...

- If you have HTTP/HTTPS -- Use the router
- If you have Encrypted TCP (i.e. TLS with the SNI header) -- Use the router
- If you have TCP or UDP
  - Use the new bare-metal LoadBalancer, but that limits you to a single ingress IP (can be a VIP, but still will be a single machine for initial load balancing).  It simplifies the admin job, but uses one IP per service.
  - Manually assign ExternalIPs to the service.  Here you can assign a set of IPs, so can have multiple machines for the incoming load balancing.  But requires elevated permissions to assign, and manual tracking of what IP:ports have been used.
 - Use NodePorts to expose the service on ALL nodes in the cluster.  More wasteful of scarce port resources.  BUT a little easier to set up multiple.  Again, requires more privilege.